packages/@ngtools/webpack/src/plugin.ts

```diff
@@ -93,7 +65,12 @@ export class AotPlugin implements Tapable {
   get compilerHost() { return this._compilerHost; }
   get compilerOptions() { return this._compilerOptions; }
   get done() { return this._donePromise; }
-  get entryModule() { return this._entryModule; }
+  get entryModule() {
+    const splitted = this._entryModule.split('#');
```

Comment:
const {path, className} = this._entryModule.split('#');
return {path, className: className || 'default'};